drivers/i2c/i2c_sam_twi.c

```diff
@@ -66,6 +66,8 @@ struct i2c_sam_twi_dev_data {
 	struct twi_msg msg;
 };
 
+#define DEV_NAME(dev) ((dev)->name)
```

Comment:
NAK on this!

Please, open an issue about it and send a PR that drop remaining DEV_.

Reply:
Alright. I was probably jumping a bit ahead to address the integration issue. I've raised issue #42996. Let me just revert this commit from the pull request...